packages/salesforcedx-vscode-core/src/commands/functions/forceFunctionStart/ForceFunctionContainerStartExecutor.ts

```diff
@@ -35,12 +35,23 @@ const forceFunctionStartErrorInfo: {
   }
 };
 
+export const BINARY_EVENT_ENUM = {
+  CONTAINER: 'container',
+  ERROR: 'error',
+  LOG: 'log',
+  PACK: 'pack'
+};
+
 export class ForceFunctionContainerStartExecutor extends ForceFunctionStartExecutor {
+  // The function binary. It must be used as a single instance. Multiple class to the getFunctionBinary
+  // function will not provide the same instance of the binary.
+  private functionsBinary: Benny | undefined;
```

Comment:
figured out in testing that this has to be the same instance of the binary across the various methods of the listeners do not work as expected.